frontend/src/components/ui/blockEditor/DirectoryViewer.jsx

```diff
@@ -400,7 +407,7 @@ export default function DirectoryViewer({
                       tooltipPosition="left"
                       hasIconOnly
                       size="md"
-                      onClick={saveChanges}
+                      onClick={(e) => saveChanges(e)}
```

Comment:
Is that what fixes the scrolling issues where it keeps snapping back?

Reply:
Nope